engine/src/main/resources/org/hibernate/validator/ValidationMessages.properties

```diff
@@ -32,3 +31,8 @@ org.hibernate.validator.constraints.URL.message                     = must be a
 org.hibernate.validator.constraints.br.CNPJ.message                 = invalid Brazilian corporate taxpayer registry number (CNPJ)
 org.hibernate.validator.constraints.br.CPF.message                  = invalid Brazilian individual taxpayer registry number (CPF)
 org.hibernate.validator.constraints.br.TituloEleitoral.message      = invalid Brazilian Voter ID card number
+
+org.hibernate.validator.constraints.pl.REGON.message = Invalid Polish Taxpayer Identification Number (REGON)
```

Comment:
Seems we don't have a Polish translations file yet. Maybe - as a separate issue -you could create one (assuming you speak the language based on the fact that you created these constraints :) )?